### PR TITLE
feat(docs): add releases button and download assets

### LINKED ISSUE #7352: databend.rs add databend download page
**Summary**
Databend releases build every day.  you cant find some information at:

https://repo.databend.rs/databend/tags.json  (AWS S3 )
https://api.github.com/repos/datafuselabs/databend/tags

```
 {
    "name": "v0.8.14-nightly",
    "zipball_url": "https://api.github.com/repos/datafuselabs/databend/zipball/refs/tags/v0.8.14-nightly",
    "tarball_url": "https://api.github.com/repos/datafuselabs/databend/tarball/refs/tags/v0.8.14-nightly",
    "commit": {
      "sha": "07bbba3a248474a9f5e9fb44e41735d7a4aecebc",
      "url": "https://api.github.com/repos/datafuselabs/databend/commits/07bbba3a248474a9f5e9fb44e41735d7a4aecebc"
    },
    "node_id": "MDM6UmVmMzAyODI3ODA5OnJlZnMvdGFncy92MC44LjE0LW5pZ2h0bHk="
  }
```
Download from s3:
https://repo.databend.rs/databend/v0.8.14-nightly/databend-v0.8.14-nightly-x86_64-unknown-linux-musl.tar.gz

Dowload from Github: 
    https://github.com/datafuselabs/databend/releases/download/v0.8.14-nightly/databend-v0.8.14-nightly-x86_64-unknown-linux-musl.tar.gz

So download databend url  logic:

```
version= tags.name
databend_latest=(databend-%s-%s.tar.gz)%(version, arch)
if from-china:
 download_base=https://repo.databend.rs/databend
else:
 download_base=https://github.com/datafuselabs/databend/releases/download/

full_url=(%s/%s/%s)%(download_base, version, databend_latest)

```

arch type:
| Arch | Os type|
|----|----|
|x86_64-unknown-linux-gnu| x86_64 ubuntu|
|x86_64-unknown-linux-musl | x86_64 general_linux eg: centos|
|x86_64-apple-darwin | x86_64 macos |
|aarch64-unknown-linux-gnu | arm ubuntu |
|aarch64-unknown-linux-musl | arm general_linux |
|aarch64-apple-darwin | arm macos |

### PR DESCRIPTION
**Pull request by Chasen-Zhang**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Add releases button and download assets.
Hover on Releases button:
image

Fixes #7352

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ function HomepageHeader() {
                   to="/doc">
                   Documentation
               </Link>
-
+              <Releases />
```
Comment (BohuTANG) [inline website/src/pages/index.tsx:30]: Please add a `Download` menu for the release download, not the `index.tsx`.
```suggestion
              <Releases />
```

Comment (BohuTANG) [reply website/src/pages/index.tsx:30]: image

**Comment by wubx** [comment]
It would be better if we add a separate download page.

**Comment by Chasen-Zhang** [comment]
ok, I will add it a separate page

**Comment by Chasen-Zhang** [comment]
image
 @wubx @BohuTANG  Separate a page by itself

File: website/src/pages/download/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      title={`Databend - Activate your Object Storage for real-time analytics`}
+      description={`A modern Elasticity and Performance Cloud Data Warehouse, activate your Object Storage(S3, Azure Blob, or MinIO) for real-time analytics`}>
+      <div className={styles.tableWarp}>
+        <div className={styles.table}>
+          <>
+            {
+              getLatest((tagName: string)=> {
+                const data = [
+                  { name: `databend-${tagName}-aarch64-apple-darwin.tar.gz`, tagName, osType: 'macOS (ARM, 64-bit)'},
+                  { name: `databend-${tagName}-aarch64-unknown-linux-gnu.tar.gz`, tagName, osType:'Ubuntu Linux (ARM, 64-bit)'},
```
Comment (BohuTANG) [inline website/src/pages/download/index.tsx:60]: We can remove the `gun` target, and let users easier to choose.

File: website/src/pages/download/index.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    <Layout 
+      title={`Databend - Activate your Object Storage for real-time analytics`}
+      description={`A modern Elasticity and Performance Cloud Data Warehouse, activate your Object Storage(S3, Azure Blob, or MinIO) for real-time analytics`}>
+      <div className={styles.tableWarp}>
+        <div className={styles.table}>
+          <>
+            {
+              getLatest((tagName: string)=> {
+                const data = [
+                  { name: `databend-${tagName}-aarch64-apple-darwin.tar.gz`, tagName, osType: 'macOS (ARM, 64-bit)'},
```
Comment (BohuTANG) [inline website/src/pages/download/index.tsx:59]: It would be better to make the order like:
linux
...
apple

**Comment by Chasen-Zhang** [comment]
image
I have removed GNU target and reordered it.

**Comment by BohuTANG** [comment]
Thanks, can we get the target file size and put them on the table?

File: website/src/pages/download/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  ];
+  return (
+    <Layout 
+      title={`Databend - Activate your Object Storage for real-time analytics`}
+      description={`A modern Elasticity and Performance Cloud Data Warehouse, activate your Object Storage(S3, Azure Blob, or MinIO) for real-time analytics`}>
+      <div className={styles.tableWarp}>
+        <div className={styles.table}>
+          <>
+            {
+              getLatest((tagName: string)=> {
```
Comment (Xuanwo) [inline website/src/pages/download/index.tsx:55]: I prefer to generate the link during build time.

**Comment by Xuanwo** [comment]
image

**Comment by Chasen-Zhang** [comment]
I solved the loading latency problem in other ways. "generating the link during build time" I don't have the perfect solution yet, Let's just push that up for now.  Let me think about it a little bit.

**Comment by Chasen-Zhang** [comment]
You can merge to main branch first.

File: website/src/utils/tools.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+export function bytesToSize(bytes: number) {
```
Comment (BohuTANG) [inline website/src/utils/tools.ts:1]: Use https://www.npmjs.com/package/pretty-bytes or another package is better?

Comment (Chasen-Zhang) [reply website/src/utils/tools.ts]: ok, This library is good

Comment (Chasen-Zhang) [reply website/src/utils/tools.ts]: wait a moment, this library inaccuracy

**Review by BohuTANG — APPROVED** [review]
💯

**Pull request merged**